pkg/services/datasources/cache.go

```diff
@@ -31,7 +31,8 @@ func (dc *CacheServiceImpl) Init() error {
 	return nil
 }
 
-func (dc *CacheServiceImpl) GetDatasource(datasourceID int64, user *models.SignedInUser, skipCache bool) (*models.DataSource, error) {
+func (dc *CacheServiceImpl) GetDatasource(datasourceID int64, user *models.SignedInUser, skipCache bool) (
+	*models.DataSource, error) {
```

Comment:
@sakjur Oh yeah, the idea was to not make the line longer than 100 chars. Do you know if we have any rule about this? I think there's some linter can object to lines longer than a certain length, but don't recall exactly.

Reply:
:+1:

I don't think we have any strict rule about it. I remember seeing Go core devs discourage from having a particular line length limit enforced elsewhere (which is why it isn't in `go fmt`), I think with the rationale that it's better if that's something that's left to the judgment of the developer + reviewer since what is legible for a particular function might be different from function to function.

The most common pattern when this happens I've seen is to do this:

```suggestion
func (dc *CacheServiceImpl) GetDatasource(
	datasourceID int64,
	user *models.SignedInUser,
	skipCache bool,
) (*models.DataSource, error) {
```